tests/phpunit/Unit/Graph/GraphFormatterTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace SRF\Tests\Unit\Formats;
+
+use SRF\Graph\GraphFormatter;
+use SRF\Graph\GraphNode;
+
+
+/**
+ * @covers \SRF\Graph\GraphFormatter
+ * @group semantic-result-formats
+ *
+ * @license GNU GPL v2+
+ * @since 3.1
```

Comment:
wrong, just drop the since tag

Reply:
@see: https://github.com/RacoonDev/SemanticResultFormats/commit/10eb56e4c1a888dd4f9988427e55bf3f12f98917